invenio_audit_logs/records/templates/os-v1/datastream-log-v1.0.0.json

```diff
@@ -0,0 +1,31 @@
+{
+  "index_patterns": ["__SEARCH_INDEX_PREFIX__auditlog*"],
+  "data_stream": {},
+  "template": {
+    "mappings": {
+      "dynamic": "strict",
+      "numeric_detection": false,
+      "properties": {
+        "log_id": { "type": "keyword" },
```

Comment:
minor: why do we rename `id` to `log_id`? If it's for the datastreams requirement that documents don't need `_id`s, I don't think it's worth propagating across the rest of the datamodel definitions